crates/tracing-instrumentation/Cargo.toml

```diff
@@ -9,6 +9,11 @@ publish = false
 
 [features]
 default = []
+# if enabled, each "crate" will show up
+# as it's own service in collectors like `jaeger`
+# if not (default) all traces will instead
+# show up under one service `restate-server`
+service_per_crate = []
```

Comment:
I imagine this is mainly useful for Restate engineers looking at server traces, hence the feature gate? Is there much downside other than the visual appearance to enabling this by default? (Sorry for the noob question.)

Reply:
Beside the extra work (that is not very cheap) that has to be done before emitting the traces to the collectors, it also very distracting in the overview page of jaeger because each `crate` in the repo appears as it's own service 

![image (3)](https://github.com/user-attachments/assets/b6f434b9-3783-492d-b268-c74d78fa8505)

My and @tillrohrmann thought this is too much and we should go back to a single binary view. But to not completely lose it i decided to put it behind a feature and make it disabled by default.

I think your idea of having a `service` per node will look cleaner and less distracting.